woocommerce/woocommerce-ios
# Blaze: Remove feature flag for the native campaign creation flow

Closes: #13321 
Also closes #13358 


## Description

The native campaign creation flow for Blaze has been released for a few months, so this PR removes the feature flag and code related to the legacy web flow.

Also remove the unused feature flag `subscriptionProducts`.

## Testing steps

- Log in to a store eligible for Blaze.
- Start campaign creation by selecting the Promote with Blaze from an existing product / from the dashboard card / from the campaign list in Hub menu.
- If the store has no campaign yet, confirm that the correct intro view is displayed.
- After selecting a product to promote, confirm that the native creation flow is displayed. 

## Screenshots

Intro view | Native creation flow
--- | ---
<img src="https://github.com/user-attachments/assets/7853201b-fa33-4ea2-a35a-f2579b1ba57c" width=320 /> | <img src="https://github.com/user-attachments/assets/f2345501-c42c-43c0-8409-35f2c3a460cc" width=320 />


---
